lang/python/django-formtools/Makefile

```
@@ -7,7 +7,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=django-formtools
 PKG_VERSION:=2.1
-PKG_RELEASE:=4
+PKG_RELEASE:=5
 
 PKG_MAINTAINER:=Eneas U de Queiroz <cote2004-github@yahoo.com>
```

While at it, would you please update my email:
```suggestion
PKG_MAINTAINER:=Eneas U de Queiroz <cotequeiroz@gmail.com>
```
BTW, the old one will remain valid, I'm changing it because I got tired of DMARC headers being added when I post something to the mailing list.